src/mds/ScatterLock.h

```diff
@@ -45,8 +46,11 @@ class ScatterLock : public SimpleLock {
     DIRTY            = 1 << 10,
     FLUSHING         = 1 << 11,
     FLUSHED          = 1 << 12,
+    NUDGING          = 1 << 13,
```

Comment:
the 'nudging' state is similar to 'flushing' state.

Reply:
Yes, they are similar, but slightly different.

For example, when doing nudge, it may have to wait for the inode to "unfreeze" or "singleauth", during which the lock would still be dirty, not flushing, but we still have to mark it, so that other rstat flush operation can find it if the lock is what they need, too